extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/user/UserMapper.java

```diff
@@ -33,10 +34,82 @@ public interface UserMapper extends ModeledDirectoryObjectMapper<UserModel> {
      *
      * @param username
      *     The username of the user to return.
+     * 
+     * @param caseSensitive
+     *     True if the search should evaluate usernames in a case-sensitive
+     *     manner, otherwise false.
      *
      * @return
      *     The user having the given username, or null if no such user exists.
      */
-    UserModel selectOne(@Param("username") String username);
+    UserModel selectOne(@Param("username") String username,
+                        @Param("caseSensitive") boolean caseSensitive);
+    
+    /**
+     * Selects all objects which have the given identifiers. If an identifier
```

Comment:
Aren't these just users? Why be vague and call them objects? Unless I'm misunderstanding something here.

Reply:
Okay, I think I've fixed this - it was copied from the class it used to inherit it from, so I just failed to change the wording.